qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -238,7 +238,25 @@ def selection(self, html=False):
         return self._widget.selectedText()
 
     def follow_selected(self, *, tab=False):
-        log.stub()
+        def _follow_selected_cb(js_elem):
```

Comment:
I'd prefer having this as a method on the same level as `follow_selected`, for readability.